package.json

```diff
@@ -43,6 +44,7 @@
     "dexie": "^3.0.2",
     "dotenv": "^8.2.0",
     "hashicon": "^0.3.0",
+    "js-sha256": "^0.9.0",
```

Comment:
You could use this one here...no need for additional dependency

https://burst-apps-team.github.io/phoenix/modules/crypto.html#hashsha256

Reply:
can i input an bytearray somehow? i looked at it but it takes a string

Reply:
https://burst-apps-team.github.io/phoenix/modules/util.html#convertbytearraytostring